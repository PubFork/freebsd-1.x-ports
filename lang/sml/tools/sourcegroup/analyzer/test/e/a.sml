signature A = sig
  type t
  val f:unit -> t
 end

signature GS = sig
  structure AS:A
  val x:AS.t
 end

functor G (AW:A) :GS = struct
  structure AS:A=AW
  val x:AW.t = AS.f()
 end

funsig FS (AP:A) = GS

functor AA :FS = G
