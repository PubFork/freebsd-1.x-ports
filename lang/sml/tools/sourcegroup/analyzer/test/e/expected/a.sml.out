source sml "e/a.sml"
  export signature A GS 
  export functor AA G 
  export fsig FS ;

